### ISSUE TITLE
payment method

### ISSUE THREAD

**Issue by kreossino** [e107inc/vstore#19]
could you insert an alternative method of payment 
a simple IBAN bank transfer payment ? 
merchandising in pending

**Comment by CaMer0n** [comment]
@kreossino If you can find such a plugin within the omnipay package, I can try to add it.

**Comment by Moc** [comment]
I'm interested in this as well (also because of an integration of this payment method in another plugin) and am planning to look into developing this as of early January (once I get some more free time).

**Comment by CaMer0n** [comment]
Looks like it's available: https://github.com/markdegrootnl/omnipay-ideal 
I would have no way of testing it though, but it should be relatively easy to add to the vstore.

**Comment by Moc** [comment]
Yeah iDeal would be easy, but this is only for Dutch banks. Still would like to see it added 👍 
I think there's no easy way or a European solution now.

**Comment by CaMer0n** [comment]
Is this useful to you? https://github.com/andypieters/omnipay-paynl

**Comment by Moc** [comment]
This would be a third-party provider (like an intermediary), which to me personally is not useful no. Instead the iDeal package you linked above (https://github.com/markdegrootnl/omnipay-ideal) would be better as it is only dependant on the banks, and no intermediaries.

**Comment by Moc** [comment]
Okay, I think using "Mollie" could be best solution here: 
- https://www.mollie.com/en/banktransfer

Package available here: https://github.com/thephpleague/omnipay-mollie
